netCDF-FortranConfig.cmake.in does not exist

Trying to build netcdf-fortran 4.5.3 as part of Homebrew with GCC 10 fails with:

```
CMake Error: File /tmp/netcdf--fortran-20200626-12044-7c8f0i/netcdf-fortran-4.5.3/netCDF-FortranConfig.cmake.in does not exist.
CMake Error at /usr/local/Cellar/cmake/3.17.3/share/cmake/Modules/CMakePackageConfigHelpers.cmake:330 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  CMakeLists.txt:1117 (configure_package_config_file)
```

This is on macOS 10.15, where we are running:

```
cmake .. -DBUILD_SHARED_LIBS=ON -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/netcdf/4.7.4_1 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DCMAKE_OSX_SYSROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -DBUILD_TESTING=OFF -DNETCDF_C_LIBRARY=/usr/local/Cellar/netcdf/4.7.4_1/lib/libnetcdf.dylib -DENABLE_TESTS=OFF
```

The exact same command worked with the previous version (but not with GCC 10, which is why we are updating).

--- a/Makefile.am
+++ b/Makefile.am
@@ -37,7 +37,7 @@ check_nc_config:
 EXTRA_DIST = README.md RELEASE_NOTES.md netcdf-fortran.pc.in	\
 F03Interfaces_LICENSE CMakeLists.txt COPYRIGHT nf-config.in	\
 config.h.cmake.in CTestConfig.cmake.in nf-config.cmake.in \
-libnetcdff.settings.in
+libnetcdff.settings.in netCDF-FortranConfig.cmake.in
 
 # Remove these generated files, for a distclean.
 DISTCLEANFILES = VERSION libnetcdf.settings

